src/test/kotlin/icfp2019/OutputTests.kt

```diff
@@ -4,78 +4,63 @@ import org.junit.jupiter.api.Assertions
 import org.junit.jupiter.api.Test
 
 class OutputTests {
+    fun getListOfAction(): MutableList<Action> {
+        return mutableListOf(
```

Comment:
do we need a mutable list?

Reply:
Yup, so the other two test cases can add more actions

Reply:
they can add more actions by adding the new values to the list:
```kt
val quux = listOf("foo").plus(listOf("bar"))
```
this retains immutability everywhere